mysite/ct/middleware.py

```diff
@@ -16,5 +23,13 @@ def process_exception(self, request, exception):
                                                     kwargs={'user_id': user_id}))
             else:
                 return HttpResponseRedirect(reverse('ct:home'))
+        elif hasattr(smtplib, exception.__class__.__name__):
+            write_exception_to_log(exception)
+            return render_to_response(
+                    'ct/error.html',
+                    {'message': 'Something goes wrong with email sending. Please try again later.'},
+                    RequestContext(request)
+                                    )
         else:
             raise exception
+
```

Comment:
pep8: two blank lines at the end of file.
use `sh check-quality.sh pep8 psa` to check pep8 warnings - see README